### PR TITLE
Feature/kitosudv 3020 renaming

### PR DESCRIPTION
**Pull request by mrjsawdk**
## KITOS Pull request template
The following procedure dictates the steps needed before a Pull request can be merged into master.

- [x] **Implement**: 
      *All requirements are implemented and unit tests are green*
      
- [x] **Merge master into branch / rebase with master**: 
      *Make sure you are testing your changes and how they co-exist with the latest version of master*
      
- [x] **Green on integration**: 
      *All integration tests are green on integration*
      
- [x] **Request review**: 
      *Tag whomever you wish to review your code*
      
- [x] **Review completed**: 
      *Reviewer ticks this box when review comments have been submitted*
      
- [x] **Changes**: 
      *PR owner and reviewer agrees on which changes must be made and the changes are committed.*
      
- [x] **Merge master into branch / rebase with master**: 
      *Make sure you are testing your changes and how they co-exist with the latest version of master*
      
- [x] **Green on integration**: 
      *All integration tests are green on integration*

### CODE REVIEW

File: Core.ApplicationServices/Contract/Write/ItContractWriteService.cs [hunk 1]
```diff
@@ -617,15 +617,15 @@ private Maybe<OperationError> UpdateName(ItContract contract, string newName)
             contract.Name = newName;
             return Maybe<OperationError>.None;
         }
-        private Maybe<OperationError> UpdateContractCriticalityType(ItContract contract, Guid? criticalityTypeUuid)
+        private Maybe<OperationError> UpdateContractCriticality(ItContract contract, Guid? criticalityTypeUuid)
```
Comment (a-naskret) [inline Core.ApplicationServices/Contract/Write/ItContractWriteService.cs:620]: Remaining criticalityType

File: Core.ApplicationServices/Contract/Write/ItContractWriteService.cs [hunk 2]
```diff
@@ -617,15 +617,15 @@ private Maybe<OperationError> UpdateName(ItContract contract, string newName)
...
         {
             return _assignmentUpdateService.UpdateIndependentOptionTypeAssignment
             (
                 contract,
                 criticalityTypeUuid,
```
Comment (a-naskret) [inline Core.ApplicationServices/Contract/Write/ItContractWriteService.cs:625]: Remaining criticalityType

**Review by a-naskret — APPROVED** [review]
A few criticality types left

File: Tests.Integration.Presentation.Web/Contract/ItContractTest.cs [hunk 3]
```diff
@@ -111,8 +111,8 @@ public async Task Can_Add_CriticalityType(OrganizationRole role)
 
             //Assert
             Assert.Equal(contract.Id, contractResult.Id);
-            Assert.Equal(criticality.Id, contractResult.CriticalityTypeId);
-            Assert.Equal(criticality.Name, contractResult.CriticalityTypeName);
+            Assert.Equal(criticality.Id, contractResult.CriticalityId);
+            Assert.Equal(criticality.Name, contractResult.CriticalityName);
```
[the remark this answers is not in the document]
Comment (mrjsawdk) [reply Tests.Integration.Presentation.Web/Contract/ItContractTest.cs:115]: done 👍

File: Core.ApplicationServices/Contract/Write/ItContractWriteService.cs (hunk 1 above)
Comment (mrjsawdk) [reply Core.ApplicationServices/Contract/Write/ItContractWriteService.cs:620]: done 👍

**Pull request merged**